docs/release-notes/0.2/release-0.2.md

```diff
@@ -0,0 +1,61 @@
+# ML.NET 0.2 Release Notes
+
+We would like to thank the community for the engagement so far and helping us shape ML.NET.
+
+Today we are releasing ML.NET 0.2. This release focuses on addressing questions/issues, adding clustering to the list of supported machine learning tasks, enabling using data from memory to train models, easier model validation, and more.
+
+### Installation
+
+Supported platforms: Windows, MacOS, Linux (see [supported OS versions of .NET Core 2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for more detail)
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Release Notes
+
+Below are some of the highlights from this release.
+
+#### New machine learning tasks: clustering
+
+Clustering is an unsupervised learning task that groups sets of items based on their features. It identifies which items are more similar to each other than other items. This might be useful in scenarios such as organizing news articles into groups based on their topics, segmenting users based on their shopping habits, and grouping viewers based on their taste in movies. 
+
+ML.NET 0.2 exposes `KMeansPlusPlusClusterer` which implments K-Means++ clustering. [This test](https://github.com/dotnet/machinelearning/blob/78810563616f3fcb0b63eb8a50b8b2e62d9d65fc/test/Microsoft.ML.Tests/Scenarios/ClusteringTests.cs) shows how to use it (from [#222](https://github.com/dotnet/machinelearning/pull/222)).
```

Comment:
>#222 [](start = 264, length = 4)

I am curious. Comments of course, do this automatically. So I can just mention #222 and this link will work. Does the same apply to checked in artifacts? That is, if you just said #222, with no syntax, would it just work? I honestly don't know but it may be worthwhile to think about.